services/commentsService.js

```diff
@@ -0,0 +1,14 @@
+const { findReview } = require('../models/commentsDao')
+
+const viewReview = async (id) => {
+    try {
+        const review = await findReview(id);
+        return review
+        
+    } catch(err) {
+        console.error(err);
+    }    
```

Comment:
```suggestion
        const review = await findReview(id);
        return review
         
```
- try-catch 구문은 controller 레이어로 이동시키셔서, controller에서 터지는 에러도 핸들링 할 수 있도록 해주세요